src/guiengine/widgets/kart_stats_widget.cpp

```diff
@@ -81,15 +79,18 @@ KartStatsWidget::KartStatsWidget(core::recti area, const int player_id,
         m_children.push_back(skill_bar);
     }
 
-    m_skills[SKILL_MASS]->setValue((int)(props->getMass()/5));
+    // Scale the values so they look better
+    m_skills[SKILL_MASS]->setValue((int)
+            ((props->getCombinedCharacteristic()->getMass() - 20) / 4));
```

Comment:
It may be good to leave a little comment explaining where those numbers come from? 20 and 4 (and on lines below, 15 and 6, 30 and 20). I know they weren't detailed before, but if we're going to improve this could we might as well do it all the way